onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -193,7 +193,7 @@ TEST(MathOpTest, Add_Broadcast_0x1) {
   test.AddInput<float>("A", {}, {10.0f});
   test.AddInput<float>("B", {1}, {2.0f});
   test.AddOutput<float>("C", {1}, {12.0f});
-  test.Run();
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kNnapiExecutionProvider});  // NNAPI: Add does not support scalar input
 }
```

Comment:
Can we fake the shape being {1} instead of {} in NNAPI in order to support scalar input?